net.zgcoder.skycheckers.json

```diff
@@ -0,0 +1,136 @@
+			"name": "SDL2",
+			"buildsystem": "autotools",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libsdl.org/release/SDL2-2.0.12.tar.gz",
+					"sha256": "349268f695c02efbc9b9148a70b85e58cefbbf704abd3e91be654db7f1e2c863"
+				}
+			],
+			"cleanup": ["/bin/sdl2-config", "/include", "/lib/libSDL2.la", "/lib/libSDL2main.a", "/lib/libSDL2main.la", "/lib/libSDL2_test.a", "/lib/libSDL2_test.la", "/lib/cmake", "/share/aclocal"]
+		},
+		{
+			"name": "SDL2_mixer",
+			"buildsystem": "autotools",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.0.4.tar.gz",
+					"sha256": "b4cf5a382c061cd75081cf246c2aa2f9df8db04bdda8dcdc6b6cca55bede2419"
+				}
+			],
+			"cleanup": ["/lib/libSDL2_mixer.la", "/include"]
+		},
+		{
+			"name": "SDL2_ttf",
+			"buildsystem": "autotools",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libsdl.org/projects/SDL_ttf/release/SDL2_ttf-2.0.15.tar.gz",
+					"sha256": "a9eceb1ad88c1f1545cd7bd28e7cbc0b2c14191d40238f531a15b01b1b22cd33"
+				}
+			],
+			"cleanup": ["/lib/libSDL2_ttf.la", "/include"]
+		},
+		{
+			"name": "glu",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://mesa.freedesktop.org/archive/glu/glu-9.0.1.tar.xz",
+					"sha256": "fb5a4c2dd6ba6d1c21ab7c05129b0769544e1d68e1e3b0ffecb18e73c93055bc"
+				}
+			],
+			"cleanup": ["/include", "/lib/*.a", "/lib/*.la", "/lib/pkgconfig"]
+		},
+		{
+			"name": "glew",
+			"no-autogen": true,
+			"make-args": [
+				"GLEW_PREFIX=/app",
+				"GLEW_DEST=/app",
+				"LIBDIR=/app/lib"
+			],
+			"make-install-args": [
+				"GLEW_PREFIX=/app",
+				"GLEW_DEST=/app",
+				"LIBDIR=/app/lib"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/nigels-com/glew/releases/download/glew-2.1.0/glew-2.1.0.tgz",
+					"sha256": "04de91e7e6763039bc11940095cd9c7f880baba82196a7765f727ac05a993c95"
+				}
+			],
+			"cleanup": [
+				"/include",
+				"/lib/pkgconfig",
+				"/lib/*.a"
+			]
+		},
+		{
+			"name": "skycheckers",
+			"subdir": "linux",
+			"no-autogen": true,
+			"make-args": [
+				"RELEASEOPTS=-D_FLATPAK_BUILD",
+				"INSTALL_DATA=0"
+			],
+			"make-install-args": [
+				"DESTDIR=/",
+				"USER_DIRNAME=app",
+				"INSTALL_DATA=0"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/zorgiepoo/Sky-Checkers/releases/download/1.2/sc-snap-pak-repo.tgz",
+					"sha256": "5f73098d917dc68f472bd85b28c26451fbff612d819ca7ff97b709436c6c486d"
+				}
+			]
+		},
+		{
+			"name": "game-data",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "extra-data",
+					"filename": "sc-snap-pak-data.tgz",
+					"url": "https://github.com/zorgiepoo/Sky-Checkers/releases/download/1.2/sc-snap-pak-data.tgz",
```

Comment:
I'm not sure why you are using `extra-data` to distribute the game data, does the license not allow Flathub to redistribute it?

Reply:
The data is not all freely licensed. It's not clear to me whether or not this is necessary.

```
Non-redistributable files can be downloaded at install time using the extra-data source type
```

Reply:
As you are copyright holder, it can be assumed you grant us redistribution license. This is much better experience for flatpak users if `extra-data` is not involved.